include/mbedtls/config.h

```diff
@@ -2798,7 +2798,26 @@
 //#define MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES      50 /**< Maximum entries in cache */
 
 /* SSL options */
-//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maxium fragment length in bytes, determines the size of each of the two internal I/O buffers */
+//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maximum fragment length in bytes, determines the size of each of the two internal I/O buffers */
+
+//#define MBEDTLS_SSL_IN_CONTENT_LEN              16384 /**< Optionally override MBEDTLS_SSL_MAX_CONTENT_LEN for the inward TLS buffer only. */
+
+/**
+ * It is possible to save RAM by setting a smaller outward buffer using
+ * MBEDTLS_SSL_OUT_CONTENT_LEN, while keeping the default inward 16384 byte
+ * buffer to conform to the TLS specification.
+ *
+ * The minimum required outgoing buffer size is determined by the handshake
+ * protocol's usage. 4096 bytes is usually adequate.
```

Comment:
>  4096 bytes is usually adequate

Is a dangerous statement. Adequate for what? Different users and use cases will need more. Unless we can say when 4096 bytes are adequate, we shouldn't state this.